src/Microsoft.Sbom.Targets/SbomCLIToolTask.cs

```diff
@@ -38,6 +72,30 @@ protected override string GenerateCommandLineCommands()
     /// <returns></returns>
     protected override bool ValidateParameters()
     {
+        // Validate required args and args that take paths as input.
+        if (!ValidateAndSanitizeRequiredParams() || !ValidateAndSanitizeNamespaceUriUniquePart())
+        {
+            return false;
+        }
+
+        var eventLevel = ValidateAndAssignVerbosity();
+        SetOutputImportance(eventLevel);
         return true;
     }
+
+    /// <summary>
+    /// This method sets the standard output importance. Setting
+    /// it to "High" ensures all output from the SBOM CLI is printed to
+    /// Visual Studio's output console; otherwise, it is hidden.
+    /// </summary>
+    /// <param name="eventLevel"></param>
+    private void SetOutputImportance(EventLevel eventLevel)
+    {
+        this.StandardOutputImportance = "High";
+
+        if (eventLevel == EventLevel.Critical)
+        {
+            this.StandardOutputImportance = "Low";
+        }
+    }
```

Comment:
Should we map the Verbosity to an `EventLevel` value for the CLI tool? It looks like we only need to check if the verbosity is _Fatal_ for setting the `StandardOutputImportante` to `"Low"`

Reply:
Updated to remove the EventLevel logic for the CLI